assembly/hdp2.6.3/src/main/resources/common-services/SPLICEMACHINE/2.5.1/package/scripts/splice_install_master.py

```diff
@@ -0,0 +1,103 @@
+import sys, os, fnmatch
+from resource_management.libraries.script.script import Script
+from resource_management.core.resources import Directory
+from resource_management.core.resources.system import Execute, Link
+from resource_management.libraries.resources import XmlConfig
+from resource_management.libraries.functions import format
+from resource_management import *
+from urlparse import urlparse
+
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+class SpliceInstallMaster(Script):
+  def install(self, env):
+    import params
+    self.install_packages(env)
+    env.set_params(params)
+    self.configure(env)
+
+  def start(self, env, upgrade_type=None):
+    pass
+
+  def stop(self, env, upgrade_type=None):
+    pass
+
+  def status(self, env):
+    pass
+
+  def configure(self, env):
+    import params
+
+    hbase_user = params.config['configurations']['hbase-env']['hbase_user']
+    user_group = params.config['configurations']['cluster-env']["user_group"]
+    splicemachine_conf_dir = '/etc/splicemachine/conf'
+    hdfs_audit_spool = params.config['configurations']['ranger-splicemachine-audit'][
+       'xasecure.audit.destination.hdfs.batch.filespool.dir']
+    solr_audit_spool = params.config['configurations']['ranger-splicemachine-audit'][
+       'xasecure.audit.destination.solr.batch.filespool.dir']
+    policy_cache_dir = params.config['configurations']['ranger-splicemachine-security'][
+       'ranger.plugin.splicemachine.policy.cache.dir']
+    hdfs_audit_dir = params.config['configurations']['ranger-splicemachine-audit'][
+       'xasecure.audit.destination.hdfs.dir']
+
+    params.HdfsResource(hdfs_audit_dir,
+                        type="directory",
+                        action="create_on_execute",
+                        owner=hbase_user
+                        )
+
+    params.HdfsResource("/user/splice",
+                        type="directory",
+                        action="create_on_execute",
+                        owner=hbase_user)
+
+    params.HdfsResource("/user/splice/history",
+                        type="directory",
+                        action="create_on_execute",
+                        owner=hbase_user)
+
+    params.HdfsResource("/user/splice/spark-warehouse",
```

Comment:
Good catch. Currently spark.sql.warehouse.dir is set to '/user/splice/spark-warehouse'. Do we need to update the installation guide for CDH as well (in which spark-warehouse directory is not created)?

Reply:
In my test it seems `'/user/splice/spark-warehouse` is not needed. Deleted these code.